src/common/util/charades-generator.js

```diff
@@ -0,0 +1,21 @@
+const dictionary = require('./charades-words');
+
+const charadesWord = async () => {
+    let name = '';
+    const nameLength = 1;
+    for (let i = 0; i < nameLength; i++) {
+
+        await dictionary.random().then(word => {
```

Comment:
I think this isn't utilizing await correctly
I think it should be something like 
```
const word = dictionary.random()
name += word;
...
```

so basically I think you should use .then and an await

Reply:
`dictionary.random()` returns a promise, so wouldn't 
```
const word = dictionary.random()
name += word;
```

just create a list of promises?

Reply:
ah I understand, my bad